sampling/hierarchical/test/opentelemetry/sampling/hierarchical/sampler_test.rb

```diff
@@ -0,0 +1,168 @@
+# frozen_string_literal: true
+
+# Copyright The OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+require('test_helper')
+
+describe(OpenTelemetry::Sampling::Hierarchical::Sampler) do
+  describe('#should_sample?') do
+    it('returns DROP if it has no samplers') do
+      sampler = OpenTelemetry::Sampling::Hierarchical::Sampler.new([])
+
+      _(
+        sampler.should_sample?(
+          trace_id: SecureRandom.uuid.to_s,
+          parent_context: nil,
+          links: [],
+          name: SecureRandom.uuid.to_s,
+          kind: :internal,
+          attributes: {}
+        ).sampled?
+      ).must_equal(false)
+    end
+
+    it('returns RECORD_AND_SAMPLE if the first sampler returns RECORD_AND_SAMPLE') do
+      trace_id = SecureRandom.uuid.to_s
+      name = SecureRandom.uuid.to_s
+
+      first_sampler = Minitest::Mock.new
+      second_sampler = Minitest::Mock.new
```

Comment:
Though not required, we tend to prefer using `rspec-mocks` over `minitest` mocks due to it being a bit more feature rich.

Reply:
Oh, I didn't know. The examples I checked all used minitest. Can I resolve this point nevertheless?